compiler-plugin-tests/src/test/resources/ballerina_sources/sample_package_27/main.bal

```diff
@@ -1,5 +0,0 @@
-import ballerina/io;
-
-public function main() {
```

Comment:
Why this is removed and not removed the test case? `testRecursiveRecordDefinitionsAsReturnType`

Reply:
This is a HTTP service related test-case.This file was previously added mistakenly hence removed it.